business/apps_test.go

```diff
@@ -49,7 +49,8 @@ func TestGetAppListFromDeployments(t *testing.T) {
 	k8s.On("GetServices", mock.AnythingOfType("string"), mock.AnythingOfType("map[string]string")).Return([]core_v1.Service{}, nil)
 	svc := setupAppService(k8s)
 
-	appList, _ := svc.GetAppList(context.TODO(), "Namespace", false)
+	criteria := AppCriteria{Namespace: "Namespace", IncludeIstioResources: false}
+	appList, _ := svc.GetAppList(context.TODO(), criteria)
```

Comment:
Although I know that this is a unit test, but maybe adding an error handling is better? This is assuming that we can get app list correctly.